xonsh/built_ins.py

```diff
@@ -1468,6 +1469,8 @@ def link_builtins(self, execer=None):
             proxy = DynamicAccessProxy(refname, objname)
             setattr(builtins, refname, proxy)
 
+        builtins.printx = print_color
```

Comment:
I think this needs to be a dynamic proxy, like everything else. Also, I would prefer the name to remain the same. That is it should be `print_color` in builtins too.

Reply:
With this long name this PR loses the power.
We already have `evalx`, `execx` instead of `eval_xonsh`, `exec_xonsh`. Why we should stay here on `print_color`. 
I can suggest an alternative - `printc` instead of `printx`. If it's also not a variant I'm going to close this PR.

Reply:
Actually that is a good point that those should probably have longer names too.  People can always add shorter names in their xonshrc if they want. I am personally tending in the direction that explicit is better than implicit

Reply:
But no matter what the name it does need to be a dynamic proxy

Reply:
I understand you Pythonish-way point. But xonsh more than Python. It makes a great try to replace shell scripting. From this point the way to reinvent bycicles i.e. creating xonshrc/xontrib/common_lib to add exit/print_color every time is not looks so useful for ordinary administrator/devops (I tested this on friends!). When you just want to write some commands and print beautiful colored text and exit you don't want to import something or thinking about what path you should register to init rc/xontrib/my_awesome_buildins_lib.

Reply:
> But no matter what the name it does need to be a dynamic proxy

Thank you for your help! I've moved `printx` to proxy and also added example to the "Bash to xonsh" doc. Take a look at the doc! I'm happy to write the docs where worst things becomes beautiful! Let's add printx! It's small thing that makes a great mood while scripting.